plugin/caddyfile/testdata/labels/quotes.txt

```diff
@@ -0,0 +1,18 @@
+caddy                  = localhost
+caddy.0_with_spaces    = "a b c d"
+caddy.1_without_spaces = "abcd"
+caddy.2_multiple       = "a b c d" "abcd"
+caddy.3_back           = `{"some":"json"}`
+caddy.4_escaped        = "a\"b"
+caddy.5_unbalanced     = "a
+caddy.6_unbalanced     = a"
+----------
+localhost {
+	with_spaces "a b c d"
+	without_spaces abcd
+	multiple "a b c d" abcd
+	back `{"some":"json"}`
+	escaped `a"b`
```

Comment:
@francislavoie 
This is now a case, where we would not preserve the original quotes used to declare the token.
But the generator knows backticks generate a more readable string because it doesn't require scaping.

Reply:
@francislavoie 
What do you think?

Reply:
I guess I'm indifferent? Feels like an edgecase that won't come up so often. :man_shrugging: